src/raftstore/store/peer.rs

```diff
@@ -1333,6 +1349,9 @@ impl Peer {
         cmd_resp::bind_term(&mut resp, self.term());
         cb.call_box((resp,));
 
+        let d = clocktime::raw_now() - lease_renew_time;
+        request_process_time.observe(duration_to_sec(d.to_std().unwrap()) as f64);
```

Comment:
/cc @BusyJay 

I think it is better to consider the request process time in async-apply, now only wait_time is ok.